src/components/ComponentFactory.js

```diff
@@ -180,8 +195,9 @@ const ComponentFactory = (props) => {
       return null;
     }
 
-    if (domain === 'landing') {
-      return <LandingComponentFactory {...props} />;
+    // Warn on unexpected usage of domains, but don't break
+    if (domain && !(domain === 'mongodb' || domain === 'std')) {
+      console.warn(`Domain '${domain}' not yet implemented ${name ? `for '${name}'` : ''}`);
```

Comment:
```suggestion
    if (domain && !['mongodb','std'].includes(domain) {
      console.warn(`Domain '${domain}' not yet implemented ${name ? `for '${name}'` : ''}`);
```
Little nitpick to make this logic block more extensible if we ever add more domains
could also do something like
```
validDomains = ['mongodb','std']'
!validDomains.includes(domain)
```